framework/post_processors/PostProcessor.h

```diff
@@ -0,0 +1,103 @@
+#ifndef PIPER_POSTPROCESSOR_H
```

Comment:
why is this not just POSTPROCESSOR_H ?

Reply:
I have no idea how PIPER got in there. Probably because I tested it there.

Anyways, The convention is <ProjectName>_<Filename>_H. Of course all caps. Additionally, if there is a duplicate filename you can add more here to distinguish them.

When you use CLION to create a header file it automatically does it for you.